core/contracts/common/implementation/TestnetERC20.sol

```diff
@@ -9,10 +9,17 @@ import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
  * this token should never be used to store real value since it allows permissionless minting.
  */
 contract TestnetERC20 is ERC20 {
+    string public name;
```

Comment:
I had to modify this contract (and it's migration) to enable non-18 delimited tokens.